packages/eslint-plugin-template/src/rules/accessibility-table-scope.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  createESLintRule,
+  getTemplateParserServices,
+} from '../utils/create-eslint-rule';
+
+type Options = [];
+export type MessageIds = 'accessibilityTableScope';
+export const RULE_NAME = 'accessibility-table-scope';
+
+export default createESLintRule<Options, MessageIds>({
+  name: RULE_NAME,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Ensures that scope is not used on any element except th.',
+      category: 'Best Practices',
+      recommended: false,
+    },
+    schema: [],
+    messages: {
+      accessibilityTableScope: 'Scope attribute can only be on <th> element.',
+    },
+  },
+  defaultOptions: [],
+  create(context) {
+    const parserServices = getTemplateParserServices(context);
+
+    return {
+      Element(node: any) {
```

Comment:
Same feedback as previous PR regarding visitor keys

Reply:
Ok, will update it after the previous PR gets merged and rebasing.

Reply:
Great, thanks!